docs/source/howto/installation.rst

```diff
@@ -26,23 +26,271 @@ Configuring your installation
 
 
 .. _how-to:installation:backup:
 
 Backing up your installation
 ============================
 
-`#4004`_
+AiiDA stores:
+
+ * the profile configuration in the ``config.json`` file located in the ``.aiida`` folder.
+   Typically located at ``~/.aiida`` (see also :ref:`configure_aiida`).
+
+ * files associated with nodes in the repository folder (one per profile). Typically located in the ``.aiida`` folder.
+
+ * queryable metadata in the PostgreSQL database (one per profile).
+
+All three components are required for the operation of AiiDA and should be backed up.
+
+.. _how-to:installation:backup:repository:
+
+Repository backup (``.aiida`` folder)
+-------------------------------------
+
+For **small repositories** back up the ``.aiida`` folder either by making a full copy or using tools for incremental backups like ``rsync``.
+
+For **large repositories** with more than 100k nodes, even incremental backups can take a significant amount of time.
+AiiDA provides a helper script that takes advantage of the AiiDA database in order to figure out which files have been added since your last backup.
+The instructions below explain how to use it:
+
+
+ 1. Configure your backup using ``verdi -p PROFILENAME devel configure-backup`` where ``PROFILENAME`` is the name of the AiiDA profile that should be backed up.
+    This will ask for information on:
+
+    * The "backup folder", where the backup *configuration file* will be placed.
+      This defaults to a folder named ``backup_PROFILENAME`` in your ``.aiida`` directory.
+
+    * The "destination folder", where the files of the backup will be stored.
+      This defaults to a subfolder of the backup folder but we **strongly suggest to back up to a different drive** (see note below).
+
+    The configuration step creates two files in the "backup folder": a ``backup_info.json`` configuration file (can also be edited manually) and a ``start_backup.py`` script.
+
+    .. note::
+
+        Using the same disk for your backup forgoes protection against the most common cause of data loss: disk failure.
+        One simple option is to use a destination folder mounted over ssh.
+
+        On Ubuntu, install ``sshfs`` using ``sudo apt-get install sshfs``.
+        Imagine you run your calculations on `server_1` and would like to back up regularly to `server_2`.
+        Mount a `server_2` directory on `server_1` using the following command on `server_1`:
+
+        .. code-block:: shell
+
+            sshfs -o idmap=user -o rw backup_user@server_2:/home/backup_user/backup_destination_dir/ /home/aiida_user/remote_backup_dir/
+
+        Use ``gnome-session-properties`` in the terminal to add this line to the actions performed at start-up.
+        Do **not** add it to your shell's startup file (e.g. ``.bashrc``) or your computer will complain that the mount point is not empty whenever you open a new terminal.
```

Comment:
Ok, I'll use the panels (they will appear once #4107 is merged as well, the only needed change from there is to pip-install version 0.4.0 of sphinx_panels instead of 0.1.0)